hackathon_site/event/views.py

```diff
@@ -1,21 +1,29 @@
 from datetime import datetime, timedelta
 
-from django.conf import settings
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.db import transaction
 from django.shortcuts import redirect
 from django.urls import reverse_lazy
 from django.views.generic.base import TemplateView
 from django.views.generic.edit import FormView
 
-from rest_framework import generics, mixins
+
+from django.conf import settings
+from django_filters import rest_framework as filters
+
 
 from hackathon_site.utils import is_registration_open
 from registration.forms import JoinTeamForm
 from registration.models import Team as RegistrationTeam
 
+from rest_framework import generics, mixins
```

Comment:
Minor (just noticed this), can you move this to below the django imports so 3rd party imports are all grouped together?